core/tests/build_sources/extensions/sample.spec.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 The Oppia Authors. All Rights Reserved.
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
```

Comment:
Made this into a generic TS test file since that's all the corresponding test actually needs (note that this entire file is just test data).